src/olympia/constants/applications.py

```diff
@@ -160,6 +164,7 @@ class UNKNOWN_APP(App):
 APP_IDS = {app.id: app for app in APP_USAGE}
 APP_GUIDS = {app.guid: app for app in APP_USAGE}
 APPS_CHOICES = tuple((app.id, app.pretty) for app in APP_USAGE)
+APP_USAGE_STATICTHEME = (FIREFOX,)
```

Comment:
I would put that right below `APP_USAGE` for some reason. Static themes are not yet supported on android?

Reply:
They're not, and potentially they may never be.  (Still under decision by the Fennec people)

Reply:
https://github.com/mozilla/addons-server/pull/6801/commits/13edb550e271340c49c4fb00fdfc946362b1a12c